web/src/settings_realm_user_settings_defaults.js

```diff
@@ -53,8 +54,24 @@ export function update_page(property) {
 
 export function set_up() {
     const $container = $(realm_default_settings_panel.container);
-    settings_display.set_up(realm_default_settings_panel);
+    const $notification_sound_elem = $("#realm-default-notification-sound-audio");
+    const $notification_sound_dropdown = $container.find(".setting_notification_sound");
+
+    notifications.update_notification_sound_source(
+        $notification_sound_elem,
+        realm_default_settings_panel.settings_object,
+    );
+
+    $notification_sound_dropdown.on("change", () => {
+        const sound = $notification_sound_dropdown.val().toLowerCase();
+        realm_default_settings_panel.notification_sound = sound;
```

Comment:
Do we need this line? I cannot see where this is used.

Reply:
You are right. This line is redundant. I have removed it now.